src/pineko/configs.py

```diff
@@ -47,16 +57,11 @@ def enhance_paths(configs_):
         configuration
     """
     # required keys without default
-    for key in [
-        "ymldb",
-        "operator_cards",
-        "grids",
-        "theory_cards",
-        "fktables",
-        "ekos",
-    ]:
+    for key in needed_keys:
         if key not in configs_["paths"]:
             raise ValueError(f"Configuration is missing a 'paths.{key}' key")
+        if key == "operator_card_template_name":
```

Comment:
the string `operator_card_template_name` is spelled to many times ... can't you refactor? e.g. a list of required dirs and a list of required files?

Reply:
You mean, we should have something similar to what we have for needed folders but for files, right? We can have but at the moment the needed file list would only contain one entry

Reply:
In general, any string should appear in the package exactly once. If you use many times, assign to a constant (same for any other value).

Reply:
the list is maybe one element at this point in time, but it might grow ...

Reply:
Ok I agree, done :)